packages/core/src/render3/ng_module_ref.ts

```diff
@@ -40,6 +33,7 @@ export class NgModuleRef<T> extends viewEngine_NgModuleRef<T> implements Interna
   injector: Injector = this;
   instance: T;
   destroyCbs: (() => void)[]|null = [];
+  readonly componentFactoryResolver: ComponentFactoryResolver;
```

Comment:
Why not just do this?

```suggestion
  readonly componentFactoryResolver: ComponentFactoryResolver = new ComponentFactoryResolver(this);
```